public/Lab-Files/Lesson02/Challenges/blog-post.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Blog</title>
+</head>
+<body>
+<h1>
+    Bert's Web Blog Post
```

Comment:
There are no errors in your code at all! Great job! 
I would suggest lines 9 and 10 to just go in line 8, like so:
<h1> Bert's Web Blog Post </h1>.